test/test_autotag.py

```diff
@@ -1040,3 +1045,35 @@ def test_ampersand_expansion(self):
     def test_accented_characters(self):
         dist = string_dist("\xe9\xe1\xf1", "ean")
         assert dist == 0.0
+
+
+@pytest.mark.parametrize(
+    "single_field,list_field",
+    [
+        ("mb_artistid", "mb_artistids"),
+        ("mb_albumartistid", "mb_albumartistids"),
+        ("albumtype", "albumtypes"),
+    ],
+)
```

Comment:
Not savvy with pytest yet. Can you explain in 3 words what this does?

Reply:
These are test cases: it tests each `(single_field, list_field)` combination against each `(single_value, list_value)` combination.